src/main/java/com/ait/lienzo/client/core/image/ImageStrip.java

```diff
@@ -0,0 +1,69 @@
+package com.ait.lienzo.client.core.image;
+
+import com.google.gwt.resources.client.ImageResource;
+
+public class ImageStrip {
```

Comment:
Class formatted as Kie-Group class, not Lienzo's one. If we want to merge changes to upstream it will be an issue in the future.